tests/test_screendoor.py

```diff
@@ -0,0 +1,19 @@
+"""Tests for screendoorapi package"""
+from screendoor_sdk.screendoor import Screendoor
+
+def test_screendoor_url():
+    """Test screen api url"""
+    sd_key = 'SCREENDOOR_API_KEY'
```

Comment:
Shouldn't these values be coming from the .env file?
How do the values in the .env file become environment vars?  Is that handled automatically by the falcon framework?

Reply:
This test is to test the formation of the URL, verifying it is constructing the proper URL to screendoor's API.   In the examples folder, there is a sample usage of reading off environmental variables.

Reply:
I see the code that reads the environment var.
`scrndr = Screendoor(os.environ['SD_KEY'])`
I'm asking about how the environment vars get set from .env file.  I don't see anything that runs something like `source .env`.  Is that done by falcon?

Reply:
I don't think falcon handles that for you, so its up to you to decide how you want to load in environmental variables. there are many options:
* run commands like `source .env` in your environment (as you mentioned)
* use a library like `python-dotenv`
* configure in your PaaS solution. I believe both CircleCI and Heroku has the ability to do this. 
* other?

Reply:
Since this repo is just for the SDK itself and not an actual "app". It probably shouldn't force people to do it one way or other.

Reply:
Thanks for answering my question. 👍